lib/OpenQA/WebAPI/Controller/API/V1/JobTemplate.pm

```diff
@@ -21,6 +21,9 @@ use JSON::Validator;
 use Text::Diff;
 use Storable 'dclone';
 
+# Prevent find_or_create from producing a warning because of nullable columns in constraints
+$ENV{DBIC_NULLABLE_KEY_NOWARN} = 1;
```

Comment:
environment variables really don't belong in random controllers.

Reply:
As discussed yesterday: I consider it wrong to disable this warning and would switch to "" for empty name.

Reply:
Ack. I couldn't change the status back to draft, but I'm pushing an update based on our conversation in a jiffy.